src/Microsoft.AspNetCore.Razor.Language/Legacy/AddTagHelperChunkGenerator.cs

```diff
@@ -10,14 +10,36 @@ namespace Microsoft.AspNetCore.Razor.Language.Legacy
 {
     internal class AddTagHelperChunkGenerator : SpanChunkGenerator
     {
-        public AddTagHelperChunkGenerator(string lookupText, List<RazorDiagnostic> diagnostics)
+        public AddTagHelperChunkGenerator(
+            string lookupText,
+            string directiveText,
+            SourceLocation directiveStart,
+            string assemblyName,
+            string typePattern,
+            SourceLocation assemblyNameLocation,
+            List<RazorDiagnostic> diagnostics)
         {
             LookupText = lookupText;
+            DirectiveText = directiveText;
+            DirectiveStart = directiveStart;
+            AssemblyName = assemblyName;
+            TypePattern = typePattern;
+            AssemblyNameLocation = assemblyNameLocation;
             Diagnostics = diagnostics;
         }
 
         public string LookupText { get; }
```

Comment:
Just realized this is being used in the IR nodes to provide intellisense. So can't nuke this.